frontend/src/app/DeploySmokeTest.test.tsx

```diff
@@ -10,21 +16,38 @@ describe("DeploySmokeTest", () => {
 
   it("renders success when returned from the API endpoint", async () => {
     (fetch as FetchMock).mockResponseOnce(JSON.stringify({ status: "UP" }));
+    (fetch as FetchMock).mockResponseOnce(
+      JSON.stringify({ components: { okta: { status: "UP" } }, status: "DOWN" })
+    );
 
     render(<DeploySmokeTest />);
     await waitFor(() =>
-      expect(screen.queryByText("Status loading...")).not.toBeInTheDocument()
+      expect(screen.queryByText(APP_STATUS_LOADING)).not.toBeInTheDocument()
     );
-    expect(screen.getByText("Status returned success :)"));
+    expect(screen.getByText(APP_STATUS_SUCCESS));
   });
 
   it("renders failure when returned from the API endpoint", async () => {
-    (fetch as FetchMock).mockResponseOnce(JSON.stringify({ status: "DOWN" }));
+    (fetch as FetchMock).mockResponse(
+      JSON.stringify({ components: { okta: { status: "UP" } }, status: "DOWN" })
```

Comment:
I think this test would be easier to understand if the shape of the mocked response matched the shape of the response from `/actuator/health/backend-and-db-smoke-test`, since that's what we're using to determine what app status text to show